### PR TITLE
Move #packit to Libera.Chat

### PR DESCRIPTION
**Pull request by csomh**
Signed-off-by: Hunor Csomortáni <csomh@redhat.com>

### CODE REVIEW

File: packit_service/worker/build/build_helper.py [hunk 1]
```diff
@@ -383,7 +383,7 @@ def _create_srpm(self):
 
             srpm_logs += (
                 f"\nMessage: {exception}\nException: {exception!r}\n{self.msg_retrigger}"
-                "\nPlease join the freenode IRC channel #packit for the latest info.\n"
+                "\nPlease join #packit on irc.libera.chat for the latest info.\n"
```
Comment (csomh) [inline packit_service/worker/build/build_helper.py:386]: @lachmanfrantisek This message seems a little bit vague to me (ie: latest info on what?). What was the original intention here? I'm looking for a way to improve this.

Comment (TomasTomecek) [reply packit_service/worker/build/build_helper.py:386]: I guess the intention was to get the latest information about the current status of the deployment: check /topic in the channel or latest messages. Mainly to know if there are any issues in the production env and the ETA to resolve them.

Comment (csomh) [reply packit_service/worker/build/build_helper.py]: I've updated this to "Please join #packit on irc.libera.chat if you need help with the error above.", which covers the service-status aspect, but also more, and maybe it's less confusing. Let me know what you think about it!

**Review by TomasTomecek — APPROVED** [review]
Nice and clear!

File: packit_service/worker/build/build_helper.py (hunk 1 above)
Comment (lachmanfrantisek) [reply packit_service/worker/build/build_helper.py]: @csomh Sorry, I've missed the notification.

I'm worried I'm not the original author of that message (price for refactoring..;), but agree with your improvement.

**Pull request merged**